src/pages/components/progress-bar/usage.mdx

```diff
@@ -0,0 +1,504 @@
+
+A progress bar provides feedback about the duration and progression of a
+process.
+
+</PageDescription>
+
+<AnchorLinks>
+
+<AnchorLink>Overview</AnchorLink>
+<AnchorLink>Live demo</AnchorLink>
+<AnchorLink>Variants</AnchorLink>
+<AnchorLink>Formatting</AnchorLink>
+<AnchorLink>Content</AnchorLink>
+<AnchorLink>Universal behaviors</AnchorLink>
+<AnchorLink>Modifiers</AnchorLink>
+<AnchorLink>Accessibility</AnchorLink>
+<AnchorLink>Related</AnchorLink>
+<AnchorLink>References</AnchorLink>
+<AnchorLink>Feedback</AnchorLink>
+
+</AnchorLinks>
+
+## Overview
+
+A progress bar indicates that the user's request has been received and the
+application is making progress toward completing the requested action. Progress
+bars inform users about the status of ongoing processes, the estimated time of
+how long a process will take, or a request is being excuted.
+
+<Row>
+<Column colLg={12}>
+
+![Progress bar hero image](images/progress-bar-usage-1.png)
+
+</Column>
+</Row>
+
+#### When to use
+
+- If a process can take a considerable or unknown amount of time
+- When the process can be described with quantitative information, for example
+  as a percentage
+- To visually show the progression of a system operation such as downloading,
+  uploading, loading data, submitting a form, or saving updates
+- To convey that data is being requested, transferred or processed
+
+#### When not to use
+
+- When the progress bar will be replaced by a minor user interface element once
+  loaded, use [skeleton states](/patterns/loading) instead
+- When manual user actions are required to progress, use a
+  [progress indicator](/components/progress-indicator/usage) instead
+- When the progress is determined by user actions such as tutorials completed,
+  or storage space, rather than system actions
+- If the process takes less than 10 seconds to load, use
+  [loading spinners](/components/loading/usage) instead
+
+#### Loading, inline loading, or progress bar?
+
+- If an activity displays a loading spinners on one screen, it should not show
+  the progress bar elsewhere in the design.
+- Both progress bars and circles can show either determinate or indeterminate
+  progress. The given area should help determine if a progress bar or circle is
+  best for that space. Progress bars are preferred in vertically narrow areas
+  (tables, cards, etc.) while progress circles are ideal for full page loading
+  or in very small areas.
+
+## Live demo
+
+## Variants
+
+A progress bar can either be determinate or indeterminate. The determinate
+variant is visualized as a track on which a bar fills up from left to right
+representing the progress. The indeterminate variant is visualized as a track on
+which a bar is moving along in constant speed, repeating over time.
+
+When it is neccesary to show the percentage and the descriptive text, the
+developers will have to calculate the percentage on their side with the value:
+`${(value/max*100).toFixed(2)}% (${value} of ${max} items)`
```

Comment:
It might be better to put this in storybook docs rather than on the website? Seems like a piece of minutia not directly related to component guidance.